packages/react-static/src/static/webpack/webpack.config.prod.js

```diff
@@ -125,7 +125,7 @@ function common(config) {
       new CaseSensitivePathsPlugin(),
       config.bundleAnalyzer && new BundleAnalyzerPlugin(),
     ].filter(d => d),
-    devtool: 'source-map',
+    devtool: undefined,
   }
```

Comment:
Why not `false`?

https://webpack.js.org/configuration/devtool#devtool

Reply:
Wasn't sure. Change it, if you like. Undefined does work too

Reply:
I thought it defines a command, so false for a command - well, great JS world :)

Reply:
While it does work in this scenario, `undefined` can sometimes denote the intent of "use the default". Let's go with `false`.